bigquery_etl/cli/monitoring.py

```diff
@@ -181,7 +181,8 @@ def deploy(
             input_path=[
                 metadata_file.parent / BIGCONFIG_FILE
                 for metadata_file in list(set(metadata_files))
-            ],
+            ]
+            + [Path(f"{sql_dir}/{project_id}/{BIGCONFIG_FILE}")],
```

Comment:
We should probably check if this file exists, since not all projects might have it.

Reply:
moved into sql_dir, should always exist there